src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -1956,6 +1958,14 @@ private static CompletionItemKind MapCompletionKind(CompletionType completionTyp
                     }
                 }
             }
+            else if (((completionDetails.CompletionType == CompletionType.File) ||
+                      (completionDetails.CompletionType == CompletionType.Folder)) &&
+                     (completionText.EndsWith("\"") || completionText.EndsWith("'")))
+            {
+                int len = completionDetails.CompletionText.Length;
+                completionText = completionDetails.CompletionText.Insert(len - 1, "$0");
```

Comment:
can you explain what this insert is doing and what the `$0` is used for?

(and maybe add a comment :) )

Reply:
Sure!